deployment/distribution/build.gradle

```diff
@@ -25,12 +25,19 @@ applicationDistribution.from('src/main/resources/') {
     into 'conf'
 }
 
+// Copy implementation examples into the distribution
 project(':external:implementations').getSubprojects().each { subProject ->
-    applicationDistribution.from(subProject.file('build/libs')) {
-        into 'implementations/' + subProject.name
+    applicationDistribution.into('implementations/' + subProject.name.replace("samlconf-", "").replace("-impl", "")) {
```

Comment:
I don't like this String replacing thing, but it gets the job done. I didn't want the directories in the distribution to be `samlconf-ddf-impl` and `samlconf-keycloak-impl`.